core/src/main/resources/error/error-classes.json

```diff
@@ -143,6 +143,11 @@
           "Offset expression <offset> must be a literal."
         ]
       },
+      "INVALID_ELEMENT_TYPE" : {
+        "message" : [
+          "Input to function <functionName> cannot contain elements of <elementType>. <message>"
```

Comment:
Let's don't pass `message`, and introduce more specific error class which includes the message.

Reply:
Done!